pkg/net/libp2p/key.go

```diff
@@ -49,11 +66,25 @@ func operatorPublicKeyToNetworkPublicKey(
 		return nil, fmt.Errorf("libp2p supports only secp256k1 operator keys")
 	}
 
-	return &libp2pcrypto.Secp256k1PublicKey{
-		Curve: DefaultCurve,
-		X:     operatorPublicKey.X,
-		Y:     operatorPublicKey.Y,
-	}, nil
+	operatorPublicKeyBytes := operator.MarshalCompressed(operatorPublicKey)
+
+	// Note that `libp2pcrypto.UnmarshalSecp256k1PublicKey` uses the secp256k1
+	// implementation provided by decred underneath
+	pubKey, err := libp2pcrypto.UnmarshalSecp256k1PublicKey(
```

Comment:
s/pubKey/libp2pPublicKey

Reply:
Done in https://github.com/keep-network/keep-core/commit/86d4e21916f151b85db250a1431977fe4ca6ab8a.